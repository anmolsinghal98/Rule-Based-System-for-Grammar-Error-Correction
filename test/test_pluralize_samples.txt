He is working on a project.
He is working on many project.
Apple are in the basket. There are some orange and some mango present as well.
There is only one books in the bag.
Here are my answers to your questions. First, I'm most interested in the sports' topics. I like sport. So, I want to see it. Second, we think it is a good idea. I want to see the news that happens around the world. I'm looking forward to see you. Good luck!!
I'm most interested in soccers topics. Also I prefer receiving the news through e - mail.
The news is always a good mode to increase the knowledge about the society.
I do not like to put sugars in my coffee.
I think, it's a goods idea!
The egg are cold and you need to warm them again before eating them. 
Foxes is a wild animals. But, lion are more dangerous. 
There are 3 book on the table. I want only one books among them. 
I'll answer your questions. First, I am most interested in sports. Second, I prefer receiving the news by e - mail. I hope my answers will help you make the newspaper better!
I think it will be important that "how will you express yourself". Robots can "think" and "progress" themselves, but they won't be expressing. Expressions can be anything. It could be your communication abilty, art, science, sports abilty or writing abilty. I think it's important because if you express, people will find you. They will think you are necessary person than robots or you will be a good affection for the company. So, now we become adults, we should learn how to express ourselves. You should experience as much as you can, and study a lot, and also read many books. Learning will show you different ways of life and different things that you will love. Now highschool days isa great stage for practicing expressions. We must use this chance and let's get a job that we want to do!!