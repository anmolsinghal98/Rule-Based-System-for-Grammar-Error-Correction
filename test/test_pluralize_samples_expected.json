[
  {
    "Sample No": 1,
    "Detections": []
  },
  {
    "Sample No": 2,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "5",
        "Word String": "many",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 3,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "Apple",
        "Error Type": "Pluralization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "3",
        "Word String": "some",
        "Error Type": "Pluralization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "6",
        "Word String": "some",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 4,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "4",
        "Word String": "one",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 5,
    "Detections": []
  },
  {
    "Sample No": 6,
    "Detections": []
  },
  {
    "Sample No": 7,
    "Detections": []
  },
  {
    "Sample No": 8,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "7",
        "Word String": "sugars",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 9,
    "Detections": []
  },
  {
    "Sample No": 10,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "2",
        "Word String": "egg",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 11,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "Foxes",
        "Error Type": "Pluralization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "3",
        "Word String": "lion",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 12,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "3",
        "Word String": "3",
        "Error Type": "Pluralization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "4",
        "Word String": "one",
        "Error Type": "Pluralization"
      }
    ]
  },
  {
    "Sample No": 13,
    "Detections": []
  },
  {
    "Sample No": 14,
    "Detections": []
  }
]